pom.xml

```diff
@@ -30,16 +30,10 @@
   <properties>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <arquillian.version>1.4.0.Final</arquillian.version>
-    <arquillian-cube.version>1.18.2</arquillian-cube.version>
-    <junit.version>4.13.2</junit.version>
-    <maven-surefire-plugin.version>2.19.1</maven-surefire-plugin.version>
-    <maven-failsafe-plugin.version>2.19.1</maven-failsafe-plugin.version>
+    <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+    <maven-failsafe-plugin.version>2.22.2</maven-failsafe-plugin.version>
     <spring-boot.version>2.4.9</spring-boot.version>
-    <ubi.version>1.3</ubi.version>
-    <fabric8.generator.from>
-      registry.access.redhat.com/ubi8/openjdk-8:${ubi.version}
-    </fabric8.generator.from>
+    <dekorate.version>2.6.0</dekorate.version>
```

Comment:
`Dekorate` version should come from the parent, the  `snowdrop-dependencies`

Reply:
The snowdrop-dependencies brings `dekorate:2.4.0` which for this repo is fine. Updated.